lib/PolarSynchronizer.js

```diff
@@ -145,6 +149,10 @@ PolarSynchronizer.prototype.ensureDirectory = function(cb) {
 }
 
 PolarSynchronizer.prototype.getLastTimeStamp = function(cb) {
+	if (this.customStartTimeStamp) {
+		cb (null, this.customStartTimeStamp);
+		return;
+	}
```

Comment:
Semantically okay, but please do not use return in if branches. Rather use if-else to represent alternatives.